transmogrifier/sources/xml/datacite.py

```diff
@@ -302,20 +246,117 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
         return fields
 
     @classmethod
-    def get_main_titles(cls, xml: Tag) -> list[str]:
+    def get_alternate_titles(
+        cls, source_record: Tag
+    ) -> list[timdex.AlternateTitle] | None:
+        alternate_titles = []
+        alternate_titles.extend(
+            [
+                timdex.AlternateTitle(
+                    value=str(title.string.strip()),
+                    kind=title["titleType"],
+                )
+                for title in source_record.find_all("title", string=True)
+                if title.get("titleType")
+            ]
+        )
+        alternate_titles.extend(list(cls._get_additional_titles(source_record)))
+        return alternate_titles or None
+
+    @classmethod
+    def _get_additional_titles(
+        cls, source_record: Tag
+    ) -> Iterator[timdex.AlternateTitle]:
+        """Get additional titles from get_main_titles method."""
+        for index, title in enumerate(cls.get_main_titles(source_record)):
+            if index > 0:
+                yield timdex.AlternateTitle(value=title)
+
+    @classmethod
+    def get_content_type(cls, source_record: Tag) -> list[str] | None:
+        content_types = []
+        if resource_type := source_record.metadata.find("resourceType"):
+            if content_type := resource_type.get("resourceTypeGeneral"):
+                if cls.valid_content_types([content_type]):
+                    content_types.append(str(content_type))
+                else:
+                    message = f'Record skipped based on content type: "{content_type}"'
+                    raise SkippedRecordEvent(
+                        message, cls.get_source_record_id(source_record)
+                    )
+        else:
+            logger.warning(
+                "Datacite record %s missing required Datacite field resourceType",
+                cls.get_source_record_id(source_record),
+            )
```

Comment:
Given that `content_type` is not a required field, I would opt to remove lines 287-291.  The warning is confusing as it is technically "not required".  This might've been related to our discussion earlier about "useful logging"? 🤔

Reply:
I agree this should go but propose we create an issue of "evaluate logging for usefulness"  to handle it globally (with other items flagged in the comments as they are encountered) and not bog down the field method refactor.  The origin of this is that `resourceType` is required by the `Datacite` schema, but we've never done anything with it and it's incongruous with how `transmogrifier` logging has developed since. `Datacite` was the first transform written and the context was the RDI project, which is no longer our context

Reply:
Ah, thank you for clarifying! Created an [issue](https://github.com/MITLibraries/transmogrifier/issues/180) for future work and discussion. :)

Reply:
Thank you for creating the issue!